echopype/echodata/convention/1.0.yml

```diff
@@ -46,9 +46,9 @@ groups:
       Only exists if complex backscatter data are already in Sonar/Beam_group1
     ep_group: Sonar/Beam_group2
   vendor:
-    name: Vendor specific
+    name: Vendor_specific
```

Comment:
Not a huge deal, but do we want an underscore here or a space? One impact that comes to mind is the repr, I believe `name` is what is currently used to set it.

Reply:
Unless something has changed, I'm pretty sure that the attribute that's critical is `ep_group`; that one does need to have the underscore. My understanding is that `name` is only used in the repr print out, and the content of the string doesn't impact anything other than the intelligibility of what users see. As for whether to use underscore or space, I think it's a toss up. I changed it to underscore just b/c for Beam_group1/2 we're also retaining the underscore.

Reply:
I suppose it is better to go with the underscore as that is the actual value used to access it in echopype.